packages/components/addon/components/hds/alert/index.js

```diff
@@ -2,6 +2,15 @@ import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
 
 export default class HdsAlertIndexComponent extends Component {
+  constructor() {
+    super(...arguments);
+
+    assert(
```

Comment:
if we want to do an assertion-based check without linking it to any internal method, we can do it in the _constructor_ of the component.